### PR TITLE
Filipino Translation Support

### PR DESCRIPTION
**Pull request by Riskitizen**
i forgor to add the last 2 lines at the first commit lmao

### CODE REVIEW

**Comment by IdaeChop** [comment]
cool

**Comment by Nv7-GitHub** [comment]
Is it up to date?

**Comment by Riskitizen** [comment]
yeah

File: eod/translation/languages/fil.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  "MarkMustHaveSeparator": "Kailangan ng \"|\" upang ipaghiwalay ang pangalan ng kategorya at ang bago nitong sulat!",
+  "PingMessage": "🏓 Pong! Latency: **%s**",
+
+  "MustHaveAdmin": "Kailangan mo ng permisong `Adminstrator`!",
+  "MustHaveAdminOrModRole": "Kailangan mo ng permisong `Administrator` o ang role na <@&%s>!",
+  "MustHaveElemForPath": "Kailangan mong makuha ang element **%s** upang makuha ang path nito!",
+  "MustHaveCatForPath": "Kailangan mong makuha ang lahat ng elements sa kategoryang **%s** upang makuha ang path nito!",
+
+  "MustHaveElemOrID": "Kailangan mong maglagay ng element o element ID!",
+  "CannotHaveBothElemAndID": "Hindi mo maaaring ilagay nang magkasabay ang element at element ID ng isang element!",
```
Comment (Nv7-GitHub) [inline eod/translation/languages/fil.json:195]: Missing `AllCats` and `Uncategorized`

**Review by Nv7-GitHub — COMMENTED** [review]
Missing some things, seems good otherwise

File: eod/translation/languages/fil.json (hunk 1 above)
Comment (Riskitizen) [reply eod/translation/languages/fil.json:195]: oh alr

Comment (Nv7-GitHub) [reply eod/translation/languages/fil.json:195]: Wait bug in scheme

**Comment by Nv7-GitHub** [comment]
Can you get latest updates to get updated schema

**Comment by Riskitizen** [comment]
fetched upstream

File: eod/translation/languages/fil.json [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
```
Comment (Nv7-GitHub) [inline eod/translation/languages/fil.json:196]: Missing `CalculatingPing`, `NewComboNews`, `NewElemNews`

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
I think you are missing a few things

File: eod/translation/languages/fil.json (hunk 2 above)
Comment (Nv7-GitHub) [reply eod/translation/languages/fil.json:196]: Also, can you rename to `tl.json` in order to fit ISO 2-letter language codes?

Comment (Riskitizen) [reply eod/translation/languages/fil.json]: ok sure

**Comment by Nv7-GitHub** [comment]
There were some more updates, could you add them?

**Comment by Riskitizen** [comment]
Supposed to be Added stuff from lines 198 to 200 + improved some translations

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**